.devcontainer/Dockerfile

```diff
@@ -0,0 +1,55 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+FROM ruby:2
```

Comment:
This is not specific enough. We don't support Ruby 2.7 yet, for example.

Reply:
Where do we specify the supported versions? Gemspec just lists > 2.4.

https://github.com/jekyll/jekyll/blob/master/jekyll.gemspec#L33

Reply:
hmmm. maybe we just shouldn't worry about it then. :smiley: